.github/pull_request_template.md

```diff
@@ -32,9 +32,9 @@ Write your release note:
 
 Format of block header: <category> <target_group>
 Possible values:
-- category:       improvement|noteworthy|action
-- target_group:   user|operator|developer
+- category:       breaking|feature|bugfix|doc|other
+- target_group:   user|operator|developer|dependency
 -->
-```improvement operator
+```bugfix operator
```

Comment:
Shouldn't the default rather be `other`?

```suggestion
```other operator
```

Reply:
I leave that up to you, I was also not 100% sure which one is the best default. If it is okay for the others, I will update change it 👍

Reply:
Seems like, `other` is good :)

Reply:
done